apps/hmc/hmc-cft-hearing-service/aat.yaml

```diff
@@ -9,3 +9,4 @@ spec:
         HMI_BASE_URL: https://hmi-apim.staging.platform.hmcts.net/hmi
         LOGGING_LEVEL_UK_GOV_HMCTS_REFORM_HMC_CONFIG: DEBUG
         LOGGING_LEVEL_UK_GOV_HMCTS_REFORM_HMC_REPOSITORY: DEBUG
+        CFT_HEARING_SERVICE_S2S_AUTHORISED_SERVICES: xui_webapp,hmc_hmi_inbound_adapter,sscs,fis_hmc_api
```

Comment:
instead of keeping the variable in all these env specific files, we can place the variable in one place  hmc-cft-hearing-service.yaml

Reply:
added in the hmc-cft-hearing-service.yaml